pkgs/development/python-modules/kubernetes-validate/default.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  pyyaml,
+  jsonschema,
+  typing-extensions,
+  importlib-resources,
+  packaging,
+  referencing,
+  flake8,
+  pytestCheckHook,
+  pep8-naming,
+  tox,
+  wheel,
+}:
+buildPythonPackage rec {
+  pname = "kubernetes-validate";
+  version = "1.31.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "willthames";
+    repo = "kubernetes-validate";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-vxsftuipw0rHQIngxKlPHwBIW+rYAjfnEEaJDKmPyfQ=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    pyyaml
+    jsonschema
+    typing-extensions
+    importlib-resources
+    packaging
+    referencing
+  ];
+
+  nativeCheckInputs = [
+    flake8
+    pytestCheckHook
+    pep8-naming
+    tox
+    wheel
```

```suggestion
    pytestCheckHook
```
We don't need to run linter or similar since we test to see if it works.